mxcube3/ui/main.jsx

```diff
@@ -70,7 +70,7 @@ export default class App extends React.Component {
   componentWillMount() {
     const persistor = persistStore(store,
            { blacklist: ['remoteAccess', 'beamline', 'sampleChanger',
-                         'form', 'login', 'general', 'logger', 'points'],
+                         'form', 'login', 'general', 'logger', 'points', 'queue'],
```

Comment:
By the way for some reason this commit appears twice (with 2 different SHA-1 ids)